content/02.main-text.md

```diff
@@ -21,6 +21,34 @@ Although we requested that some authors participate for their specific expertise
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
 
+### Collaborative writing and reviewing
+
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for more technical writing [@url:https://www.overleaf.com/ @url:https://www.authorea.com/].
+`TODO: survey other existing options, see if there are better references`
+These platforms typically offer version control and other features to support multi-author editing.
+`TODO: some use git in the back end, is that relevant?`
+However, their critical limitation for open writing is that they give all authors permission to modify all parts of the manuscript.
+This makes it difficult to maintain consistent content and tone and is susceptible to malicious edits.
```

Comment:
There is a fairly detailed version history, but I don't see anything equivalent to the GitHub contributors page.  It would be hard to trace the contributions of individual authors in the Google Doc.